react-challenge.html

```diff
@@ -35,30 +47,35 @@
 
         handleMoodOptions = () => {
           let randomIndex = Math.floor(Math.random() * this.state.moods.length )
-          this.setState({ currentMood: moods[randomIndex] })
+          this.setState({this.state.currentMood: this.state.moods[this.state.randomIndex] })
```

Comment:
```suggestion
          this.setState({currentMood: this.state.moods[randomIndex] })
```
only `moods` is a property on `this.state` so you'll only need to add it for that property